hack/subnet-clean/clean.go

```diff
@@ -0,0 +1,182 @@
+package main
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"flag"
+	"os"
+	"strings"
+	"time"
+
+	mgmtfeatures "github.com/Azure/azure-sdk-for-go/services/resources/mgmt/2019-07-01/features"
+	"github.com/sirupsen/logrus"
+
+	"github.com/Azure/ARO-RP/pkg/env"
+	utillog "github.com/Azure/ARO-RP/pkg/util/log"
+	"github.com/Azure/ARO-RP/pkg/util/purge"
+)
+
+// denylist exists as belt and braces protection for important RGs, even though
+// they may already have the persist=true tag set, especially if it is easy to
+// accidentally redeploy the RG without the persist=true tag set.
+var denylist = []string{
+	"v4-eastus",
+	"v4-australiasoutheast",
+	"v4-westeurope",
+	"management-westeurope",
+	"management-eastus",
+	"management-australiasoutheast",
+	"images",
+	"secrets",
+	"dns",
+}
```

Comment:
This list exists in clean.go now but maybe it didn't when this was first written. Can we update this to use that same list instead of duplicating it?

Reply:
Possibly -- I brought this back because one of our environments was buckling under the number of subnets, I'm not sure if we need it long-term, but since I've needed it a few times...